mephisto/abstractions/blueprints/static_html_task/source/dev/app.jsx

```diff
@@ -99,9 +96,14 @@ function MainApp() {
   );
 }
 
-function SubmitFrame({ children, onSubmit }) {
+function SubmitFrame({ children, onSubmit, currTask }) {
```

Comment:
small nit: we tend to use fully formed names in the js code by convention, so I would recommend renaming `currTask` to `currentTask`